pkg/operation/botanist/component/kubecontrollermanager/kube_controller_manager.go

```diff
@@ -581,6 +581,10 @@ func (k *kubeControllerManager) computeCommand(port int32) []string {
 		command = append(command, "--port=0")
 	}
 
+	if version.ConstraintK8sGreaterEqual119.Check(k.version) {
```

Comment:
This can cause issues if seed is a GKE cluster, a similar adaptation like [this pr](https://github.com/gardener/gardener/pull/6468) is required here.
/cc @ialidzhikov

Reply:
I don't think this is true since we don't deploy KCM into seed clusters but only shoot clusters. For the latter, the version will not have cryptic suffixes.